db/migrate/20170404110156_extend_query_model.rb

```diff
@@ -0,0 +1,5 @@
+class ExtendQueryModel < ActiveRecord::Migration[5.0]
+  def change
+    add_column :queries, :timeline_visible, :boolean, default: false
```

Comment:
We'll likely have more timeline related properties (zoom level and whatnot) soon so I don't think thats a good idea.